generators/server/files.js

```diff
@@ -605,13 +605,8 @@ const serverFiles = {
     serverMicroservice: [
         {
             condition: generator =>
-                !(
-                    generator.applicationType !== 'microservice' &&
```

Comment:
I don't remember exactly why and it's such a headache, but there was a reason why this condition was so complicated, so really not sure about this simplification.